core/src/retransmit_stage.rs

```diff
@@ -72,6 +72,8 @@ pub fn retransmit(
                 compute_retransmit_peers(DATA_PLANE_FANOUT, my_index, indexes);
             let neighbors: Vec<_> = neighbors.into_iter().map(|index| &peers[index]).collect();
             let children: Vec<_> = children.into_iter().map(|index| &peers[index]).collect();
+            turbine_start.stop();
+            turbine_total += turbine_start.as_ms();
```

Comment:
I would name it something like compute_peers or compute_shuffled_peers. turbine_total to me doesn't mean anything.

Reply:
Oh I was just trying to stick to book definitions. computing peers + shuffling them == turbine algorithm. What do you think?

Reply:
Ok, how about compute_turbine_peers then? Just turbine sounds like it could be any part of computing the peers, receiving peer packets or sending packets to peers.

Reply:
sounds good. I'll update it, still testing this out. :) fyi I also found some unnecessary locking and stuff and got rid of it via these stats. please also review the newest commit

Reply:
Yea, nice. I like it.